content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -874,7 +874,7 @@ OpenAPI v2 Publishing is available as beta since 1.15, and as alpha since 1.14.
 {{< /note >}}
 
 With the OpenAPI v2 Publishing feature enabled, CustomResourceDefinition [OpenAPI v3 validation schemas](#validation) which are [structural](#specifying-a-structural-schema) are published as part
```

Comment:
```suggestion
With the OpenAPI v2 Publishing feature enabled, CustomResourceDefinition [OpenAPI v3 validation schemas](#validation) which are [structural](#specifying-a-structural-schema) and [enable pruning](#preserving-unknown-fields) (opt-in in v1beta1, enabled by default in v1) are published as part
```

Reply:
Logically correct, but incomplete. Pruning is not required for publishing.

Reply:
it is required to get a complete schema published. remember https://github.com/kubernetes/kubernetes/pull/82666?

Reply:
Had to verify that we really went so far to require pruning. We did 🎉

Reply:
yay